iOS Example/iOS Example.xcodeproj/project.pbxproj

```diff
@@ -7,30 +7,34 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		7810985B1EAE77E900FC4CF2 /* RxSwift.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = B388AAE91C65640700726BCA /* RxSwift.framework */; };
+		7810985C1EAE77E900FC4CF2 /* RxSwift.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = B388AAE91C65640700726BCA /* RxSwift.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
```

Comment:
I'm not sure if we should be embedding the framework on the rxmediapicker level (unless this is for the demo project)

Allowing a consumer to link RxSwift seems like the better approach imo